packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
     // requires function names to match the name of the variable or property to which they are
     // assigned
     // http://eslint.org/docs/rules/func-name-matching
-    'func-name-matching': ['off', 'always', {
```

Comment:
actually, this is correct. the string option is optional, but our configs prefer to be explicit.